lib/commands/certificate.js

```diff
@@ -139,18 +141,27 @@ async function trustCertificateInPreferences (driver, name) {
     type: 'accessibility id',
     value: 'About'
   });
-  await B.delay(300);
-  await driver.mobileSwipe({
-    element: await this.findNativeElementOrElements('class name', 'XCUIElementTypeTable', false),
-    direction: 'up'
-  });
-  await clickElement(driver, {
-    type: 'accessibility id',
-    value: 'Certificate Trust Settings'
+  const switchLocator = {
+    type: '-ios class chain',
+    value: `**/XCUIElementTypeCell[\`label == '${name}'\`]/**/XCUIElementTypeSwitch`,
+  };
+  await retryInterval(5, 1, async () => {
```

Comment:
Why wait 1ms? Could just use `retry` instead of `retryInterval`.

Reply:
I forgot about retry. Will change